salt/fileclient.py

```diff
@@ -1018,6 +1035,9 @@ def get_file(self,
             else:
                 data = data['data']
             fn_.write(data)
+
+        # end of loop
```

Comment:
We don't need this comment, we can read python and know that the block has ended